### PR TITLE
improve installer and doc

### PR DESCRIPTION
**Pull request by edouard-lopez**
Please review and test before merging.

> fix some bugs, more explicit code, more colorful

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,12 +28,15 @@ omf install pure
...
 ### Manually
+
+#### via `curl`
+
 ```sh
-set -l ffns (echo $HOME/.config/fish/functions)
-curl -Ls https://raw.github.com/rafaelrinaldi/pure/master/fish_prompt.fish > $ffns/fish_prompt.fish
-. $ffns/fish_prompt.fish
+curl -Ls https://raw.github.com/rafaelrinaldi/pure/master/installer.fish > /tmp/pure_installer.fish
+source /tmp/pure_installer; and install_pure
```
Comment (oranja) [inline README.md]: source /tmp/pure_installer**.fish**; and install_pure

**Comment by edouard-lopez** [comment]
@oranja fixed
@rafaelrinaldi can you review/merge?

File: README.md [hunk 2]
```diff
@@ -28,12 +28,15 @@ omf install pure
 ```
 
 ### Manually
+
+#### via `curl`
```
Comment (rafaelrinaldi) [inline README.md]: @edouard-lopez :+1:

**Comment by rafaelrinaldi** [comment]
@edouard-lopez The install script looks awesome :sparkles: Couldn't do it any better myself.
Thanks a lot for your effort in making the project better!

**Pull request merged**